src/components/Button.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function Button({ children, className, ...props }) {
+  return (
+    <button
```

Comment:
Similar to `BaseLink`, think about adding an animation for the hover effect

Reply:
Done!